src/modules/lua/script_lua.c

```diff
@@ -136,32 +161,39 @@ static char *deny_list[] = {
     NULL,
 };
 
+static void _serverPanic(const char *file, int line, const char *msg, ...) {
+    fprintf(stderr, "------------------------------------------------");
+    fprintf(stderr, "!!! Software Failure.");
```

Comment:
```suggestion
    fprintf(stderr, "!!! Software Failure. Press left mouse button to continue.");
```

It's from Amiga. Left button doesn't make sense anymore but it's part of the original message. Do you think it's silly to keep it?

* https://en.wikipedia.org/wiki/Guru_Meditation
* https://www.generationamiga.com/2023/01/03/one-of-the-weirdest-names-for-an-error-message-but-what-is-a-guru-meditation/

**Alternative:**

Should we add a `ValkeyModule_Panic` wrapper, similar to `ValkeyModule_Assert`?

Reply:
I don't know why the mouse part of the message got dropped when I copied the code. We should keep it! I saw that message many times when I was a kid :smile: 

> Should we add a ValkeyModule_Panic wrapper, similar to ValkeyModule_Assert?

We could, but I don't think it's worth it.